src/components/pages/download/MacTab.astro

```diff
@@ -13,9 +13,39 @@ import Tab from "./Tab.astro";
 
     <Card slot="footer" kind={CardKind.Warning}>
         <p>
-            If you get a VencordInstaller can't be opened warning, right-click
-            VencordInstaller.app and click open
+            If you get a VencordInstaller can"t be opened warning, right-click
+            VencordInstaller.app and click open (macOS Sonoma and earlier)
         </p>
+        <p>
+            If you are using the developer build of macOS Seqouia, you will need
+            to scroll to the bottom of <i>Privacy & Security</i> in your System Settings,
+            and click "Open Anyway" for VencordInstaller.app
+        </p>
+        <details>
+            <summary class="seqouia-summary"
+                >Having trouble or using macOS Seqouia?</summary
+            >
+
+            <p>
+                You can allow the app to run through your System Settings, in
+                the Privacy & Settings tab. This works whether or not you are
+                using the macOS 15 Developer Build. <a
+                    href="https://support.apple.com/102445#openanyway"
+                    rel="noopener noreferrer"
+                    >Learn more
+                </a>
+            </p>
+
+            <a href="/assets/seqouia-install.png" target="_blank">
+                <img
+                    class="sequoia-install-img"
+                    width="700"
+                    height="auto"
+                    src="/assets/seqouia-install.png"
+                    alt="An image showing how to allow VencordInstaller.app to Open Anyway in Privacy & Security settings, on macOS Seqouia."
```

Comment:
you [should avoid](https://accessibility.huit.harvard.edu/describe-content-images) words like "an image" in alt texts, it's redundant